### PR TITLE
fix: Handle chunk load promise error introduced in 1.235.0 - NR-129244

### PR DESCRIPTION
**Pull request by cwli24**
Starting in 1.235.0, loading the agent in browsers with strict privacy settings resulted in an uncaught error in a promise. This error has been handled, and chunk load failures again result only in a warning.
---

### Overview

Eliminate promise rejection.

### Related Issue(s)

NR-129244 // #599 

### Testing

Verify that when one of the feature's aggregate import is blocked & fails, no uncaught promise is thrown.

### CODE REVIEW

File: src/features/session_replay/replay-mode.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ export async function getSessionReplayMode (agentId) {
     const newrelic = gosNREUM()
     // Should be enabled by configuration and using an agent build that includes it (via checking that the instrument class was initialized).
     if (getConfigurationValue(agentId, 'session_replay.enabled') && typeof newrelic.initializedAgents[agentId].features.session_replay === 'object') {
-      await newrelic.initializedAgents[agentId].features.session_replay.onAggregateImported // if Replay could not initialize, this throws a (uncaught) rejection
-      return await sharedChannel.sessionReplayInitialized // wait for replay to determine which mode it's after running its sampling logic
+      const srInitialized = await newrelic.initializedAgents[agentId].features.session_replay.onAggregateImported
+      if (srInitialized) return await sharedChannel.sessionReplayInitialized // wait for replay to determine which mode it's after running its sampling logic
```
Comment (patrickhousley) [inline src/features/session_replay/replay-mode.js:19]: What happens when `srInitialized` is `false`? This should probably have a default return.

Comment (cwli24) [reply src/features/session_replay/replay-mode.js:19]: It bleeds down into the `return MODE.OFF` like the other 2 checks

Comment (patrickhousley) [reply src/features/session_replay/replay-mode.js:19]: Not if an exception is not thrown. Right.

Comment (patrickhousley) [reply src/features/session_replay/replay-mode.js:19]: NVM i can't read.

**Review by patrickhousley — APPROVED** [review]

**Review by bjfield — APPROVED** [review]

**Pull request merged**